pkg/client/clientset_generated/clientset/clientset.go

```diff
@@ -29,11 +29,7 @@ import (
 type Interface interface {
 	Discovery() discovery.DiscoveryInterface
 	ServicecatalogV1beta1() servicecatalogv1beta1.ServicecatalogV1beta1Interface
-	// Deprecated: please explicitly pick a version if possible.
-	Servicecatalog() servicecatalogv1beta1.ServicecatalogV1beta1Interface
```

Comment:
This was finally ripped out, so I had to move everything over to using the specific version

Reply:
wow, the generated code had a warning. I'd never have seen it until now.

Reply:
super curious about the history of this, but not enough to spend more than 3 minutes on it.